source/documentation/gds_on_site_backups.md

```diff
@@ -21,7 +21,7 @@ To get on this VPN, you will need to open a support request on the GDS helpdesk.
 
 ### User Account
 
-Please speak to the RE-D team to get you set up on this.
+Please speak to the RE-Gov.Wifi team to get you set up on this.
```

Comment:
is that how we case ourselves? I thought we were GovWifi?

Reply:
Interesting question. I'd like to scrub references to RE-D because that reliability engineering team no longer exists. A bit of context: previously the RE team supporting GovWifi was called RE-D, then it was RE-Portfolio, now it's RE-GovWifi which sits in RE-GOV.UK.

When reading the docs I assumed this step would be completed by someone from GovWifi's reliability engineering support team (i.e., now RE-GovWifi). It's possible the docs are more out of date than I realised and that this step is in fact completed by someone on the GovWifi team. 

Do you happen to know if that's the case?

Reply:
I don't know but happy with ⭐️ RE-GovWifi ⭐️

Reply:
Amazing! Thanks for the spot and pointing out the errant `.`. I've updated the PR 👍